src/config/keys.ts

```diff
@@ -199,6 +199,9 @@ export const memberKeys = {
       // items liked by the member
       likedItems: [...singleBaseKey, 'likedItems'] as const,
 
+      // apps used mostly by the member
+      mostUsedApps: [...singleBaseKey, 'mostUsedApps'] as const,
```

Comment:
Woops, I told you wrong, In order to not have to send the memberId (as it can be inferred for the current member with the session, this key should be moved to the `current` section of the memberKeys below.

Reply:
Ok, thanks!